### ISSUE TITLE
NETFramework

### ISSUE THREAD

**Issue by ruslan79k** [miroiu/nodify#14]
NuGet package not installing for NETFramework

**Comment by labiej** [comment]
@miroiu 
This seems mostly a case of publishing the version of Nodify with support for multiple frameworks.

It might be worth it to consider strong name signing the assembly, see [Microsoft recommedations](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/strong-naming#create-strong-named-net-libraries) for more info. It should not break anything, although it could break things when going back.

If I have some time this week I can provide the basic implementation(without the keys of course 😉).

**Comment by miroiu** [comment]
Hi,
The problem was that the latest NuGet package at that time (v1.6.1) did not include .NET framework targets.
@ruslan79k please let me know if the new version (v1.6.2) works for you.
Sorry for the delay.

**Comment by miroiu** [comment]
@labiej 
That's a good idea. I was looking into it a while back but was afraid that I may break something.

**Comment by labiej** [comment]
@miroiu 

Sorry for not responding sooner, I've been busy.

Regarding breaking things, unless I'm missing something strong naming an assembly shouldn't break anything.
It can cause issues when the key itself changes because that will change the strong name.

I did find the time to do some more digging especially how to handle the key file itself.

* Signing an open source library is recommended by Microsoft [here](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/strong-naming#create-strong-named-net-libraries)
* It is recommended to add the 'private' key to the repository [here](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/strong-name-signing.md)
the main takeaway is that strong naming is about avoiding DLL hell, not security.

I did implement it already, when I've updated my fork I'll create a pull request.
